### PR TITLE
Detash GitHub a little

### PR DESCRIPTION
**Pull request by privat**
Drop github from model

### CODE REVIEW

File: src/api/api_auth.nit [hunk 1]
```diff
@@ -45,9 +46,10 @@ class MissionsGithubOAuthCallBack
 		if session == null then return
 		var user = session.user
 		if user == null then return
-		var player = config.players.find_by_user(user)
+		var id = "github:" + user.login
```
Comment (Morriar) [inline src/api/api_auth.nit]: `id` is used in url like `/player/:id`, can we keep them clean?

Comment (privat) [inline src/api/api_auth.nit]: ohh.. let's ignore collision then?

**Comment by Morriar** [comment]
Please rebase on master

File: src/api/api_auth.nit (hunk 1 above)
Comment (Morriar) [inline src/api/api_auth.nit]: I see 3 solutions:
- ignore collision :/
- check them and ask for the player to choose another login for `missions` (and keep an `origin` field)
- totally change the id system :(

Comment (privat) [inline src/api/api_auth.nit]: 1 is fine for the PR I think

**Comment by privat** [comment]
rerolled + removed the namespace in `_id`

File: src/api/api_auth.nit (hunk 1 above)
Comment (Morriar) [inline src/api/api_auth.nit]: I still see the prefix

**Comment by privat** [comment]
ninja reroll so you don't see the prefix

**Pull request merged**